corehq/apps/domain/auth.py

```diff
@@ -405,3 +407,19 @@ def user_can_access_domain_specific_pages(request):
         return False
 
     return couch_user.is_member_of(project) or (couch_user.is_superuser and not project.restrict_superusers)
+
+
+def connectid_token_auth(view_func):
```

Comment:
It's weird that connectid is one word. I do see that it's consistent.

Reply:
I think of `connectid` as its own thing (its a separate repo and site than connect), using one word feels like it implies that, rather than the id for connect, but I am not wedded to it.

Reply:
`connectid` and "the id for connect` being two separate concepts sounds confusing, but I don't think I have enough context on the project to be certain or to make a better suggestion.